deprecated/packages/nuget.commandline/NuGet.CommandLine.nuspec

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
+  <metadata>
+    <id>NuGet.CommandLine</id>
+    <title>[Deprecated] NuGet.CommandLine</title>
+    <version>3.4.4-rtm-final</version>
```

Comment:
This is the latest approved version currently on chocolatey.org. Actually it is not even a pre-release version but a final as the pre-release string says. Do you think it would be better to push a 3.4.4 deprecated package?